macros.in

```diff
@@ -350,7 +350,7 @@ package or when debugging this package.\
 #		"w9.gzdio"	gzip level 9 (default).
 #		"w9.bzdio"	bzip2 level 9.
 #		"w6.xzdio"	xz level 6, xz's default.
-#		"w7T16.xzdio"	xz level 7 using 16 thread (xz only)
+#		"w7T16.xzdio"	xz level 7 using 16 thread (xz and zstd only)
```

Comment:
This description is wrong. You need a new line describing this for zstd.

Reply:
I understand it that the description in parenthesis is related to the threading mode.
And `zstd` newly support it. Or am I wrong?

Reply:
`zstd` would use `w7T16.zstdio`, so it needs its own line.